npm/design-system/.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "plugins": [
+    "cypress",
+    "@cypress/dev"
```

Comment:
I wonder if we can do something like 

```json
{
  "extends": [
    "../../some-other-eslintrc"
  ]
}
```

Maybe the one in runner-ct?

Reply:
Yeah I know, there's a ton of duplication. I don't want to fix it right now. 😿